repo/meta/index.json

```diff
@@ -29,6 +72,22 @@
       }
     },
     {
```

Comment:
This is an old version of k8's. Please use the latest version. See here: https://github.com/mesosphere/universe/blob/version-1.x/repo/meta/index.json#L109